modules/fedimint-mint-client/src/oob.rs

```diff
@@ -59,6 +93,11 @@ pub struct MintOOBStatesTimeoutRefund {
     pub(crate) refund_txid: TransactionId,
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Decodable, Encodable)]
+pub struct MintOOBStatesRefundedMultiFallback {
+    pub(crate) refund_txids: Vec<TransactionId>,
+}
+
```

Comment:
After our last discussion this should be osolete

Reply:
Leftover. Deleted.